src/components/Cards/EarnCard/variants/ListItemEarnCard.tsx

```diff
@@ -43,7 +44,7 @@ export const ListItemEarnCard: FC<Omit<EarnCardProps, 'variant'>> = ({
   const items = useMemo(() => {
     const result = [];
 
-    if (apy) {
+    if (apy && apy.total) {
```

Comment:
same comment as above (skipping on apy.total == 0, and not checking tvlUSD == 0)